### DISCUSSION TITLE
Modify a deployed Landing Zone

### DISCUSSION THREAD

**Discussion by cyb3n3tic** [Azure/Enterprise-Scale#1285] [General] [answered]
If a landing zone is deployed without say Identity or Connectivity using the accelerator portal, how can you or should you add these at a later date? Would that be a use case for the bicep modules?

**Comment by jtracey93** [comment, marked as answer]
Hey @cyb3n3tic,

You could indeed use the bicep modules and the power of what-if to see what would change.

You could theoretically re-run the portal deployment entering all same same inputs as the first deployment plus the ones you want to add on, but this isn't something we have tested and support, as the ARM deployment is incremental.

Your final options is just to create the management groups manually, if you are still keen on a portal only experience, and then make the policy assignments on the management groups as defined here: https://aka.ms/alz/policies. Then move the subscriptions you desire to these management groups.

Thanks

Jack

**Reply by cyb3n3tic** [reply]
Hi Jack,

Thanks for the detailed response as always, I think I will test this out in a demo environment and see what I find just so I know for sure.
